backend/core/views.py

```
@@ -107,6 +104,8 @@
 )
 from serdes.serializers import ExportSerializer
 
+from django.core.exceptions import PermissionDenied
```

Ruff (0.8.2)

Redefinition of unused `PermissionDenied` from line 68

(F811)